src/main/java/dev/codesupport/web/api/data/repository/ShowcaseRepository.java

```diff
@@ -6,9 +6,13 @@
 
 import java.util.List;
 
+//S100 - Underscores are required spring repo syntax
+@SuppressWarnings("squid:S100")
 @Repository
 public interface ShowcaseRepository extends JpaRepository<ShowcaseEntity, Long> {
 
     List<ShowcaseEntity> findAllByUser_IdOrderById(Long userId);
 
+    List<ShowcaseEntity> findAllByUser_AliasIgnoreCaseOrderById(String alias);
```

Comment:
Codacy found an issue: [The instance method name 'findAllByUser_AliasIgnoreCaseOrderById' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=5823821)